external/llvm-project/mlir/lib/ExecutionEngine/RocmRuntimeWrappers.cpp

```diff
@@ -20,6 +20,16 @@
 
 #include "hip/hip_runtime.h"
 
+static bool isVerbose() {
+  static char *envval = std::getenv("ROCMLIR_RUNTIME_TRACE");
```

Comment:
I'd nitpick that external code should probably go for something like `MLIR_ROCM_RUNTIME_TRACE` but that's a rename we can do while upstreaming this ... come to think it, this change should probably be upstreamed right after the PR lands

Reply:
That's good since it won't collide with any ROCM defines.